cmd/routesrv/options/options.go

```diff
@@ -0,0 +1,111 @@
+	"regexp"
+	"time"
+
+	"github.com/zalando/skipper/dataclients/kubernetes"
+	"github.com/zalando/skipper/eskip"
+)
+
+type Options struct {
+	Address string
+
+	SourcePollTimeout time.Duration
+
+	OpenTracing []string
+
+	// If set makes skipper authenticate with the kubernetes API server with service account assigned to the
+	// skipper POD.
+	// If omitted skipper will rely on kubectl proxy to authenticate with API server
+	KubernetesInCluster bool
+
+	// Kubernetes API base URL. Only makes sense if KubernetesInCluster is set to false. If omitted and
+	// skipper is not running in-cluster, the default API URL will be used.
+	KubernetesURL string
+
+	// KubernetesHealthcheck, when Kubernetes ingress is set, indicates
+	// whether an automatic healthcheck route should be generated. The
+	// generated route will report healthyness when the Kubernetes API
+	// calls are successful. The healthcheck endpoint is accessible from
+	// internal IPs, with the path /kube-system/healthz.
+	KubernetesHealthcheck bool
+
+	// KubernetesHTTPSRedirect, when Kubernetes ingress is set, indicates
+	// whether an automatic redirect route should be generated to redirect
+	// HTTP requests to their HTTPS equivalent. The generated route will
+	// match requests with the X-Forwarded-Proto and X-Forwarded-Port,
+	// expected to be set by the load-balancer.
+	KubernetesHTTPSRedirect bool
+
+	// KubernetesHTTPSRedirectCode overrides the default redirect code (308)
+	// when used together with -kubernetes-https-redirect.
+	KubernetesHTTPSRedirectCode int
+
+	// KubernetesIngressClass is a regular expression, that will make
+	// skipper load only the ingress resources that have a matching
+	// kubernetes.io/ingress.class annotation. For backwards compatibility,
+	// the ingresses without an annotation, or an empty annotation, will
+	// be loaded, too.
+	KubernetesIngressClass string
+
+	// KubernetesRouteGroupClass is a regular expression, that will make skipper
+	// load only the RouteGroup resources that have a matching
+	// zalando.org/routegroup.class annotation. Any RouteGroups without the
+	// annotation, or which an empty annotation, will be loaded too.
+	KubernetesRouteGroupClass string
+
+	// PathMode controls the default interpretation of ingress paths in cases
+	// when the ingress doesn't specify it with an annotation.
+	KubernetesPathMode kubernetes.PathMode
+
+	// KubernetesNamespace is used to switch between monitoring ingresses in the cluster-scope or limit
+	// the ingresses to only those in the specified namespace. Defaults to "" which means monitor ingresses
+	// in the cluster-scope.
+	KubernetesNamespace string
+
+	// *DEPRECATED* KubernetesEnableEastWest enables cluster internal service to service communication, aka east-west traffic
+	KubernetesEnableEastWest bool
+
+	// *DEPRECATED* KubernetesEastWestDomain sets the cluster internal domain used to create additional routes in skipper, defaults to skipper.cluster.local
+	KubernetesEastWestDomain string
```

Comment:
Lets check if we need to pull this legacy stuff

Reply:
Considering it's a new binary and it was already marked `DEPRECATED` I guess we can drop it.

Reply:
Checking code, if they are really deprecated and the same can be achieved with the suggested replacement (`...EastWestRange...`)

Reply:
OK. Verified and got rid of 3 deprecated/redundant options.